apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -600,6 +592,11 @@ const WorkshopForm = React.createClass({
         validation.style.subject = "error";
         validation.help.subject = "Required.";
       }
+      if (!this.state.funded) {
+        validation.isValid = false;
+        validation.style.funded = "error";
+        validation.help.funded = "Required";
+      }
```

Comment:
Won't this fail validation whenever it's false? Shouldn't this be `this.state.funded === null` ?

Reply:
yes! i forgot to change this when we switched it away from strings. thanks!